docker-compose.yml

```diff
@@ -195,6 +204,13 @@ services:
       - ".:/home/pptruser/webknossos"
     user: ${USER_UID:-1000}:${USER_GID:-1000}
 
+  # webKnossos-connect
+  webknossos-connect:
+    image: scalableminds/webknossos-connect:master__190
+    volumes:
+      - "./conf/connect:/app/data"
```

Comment:
wkconnect saves only configuration stuff to /app/data, right? otherwise, the path on the host system should probably be named differntly

Reply:
Yes, exactly. This is only the config and the `datasets.json`